model/Core/Properties/suppliedBy.md

```diff
@@ -12,7 +12,7 @@ referenced by the Element.
 Identify the actual distribution source for the artifact (e.g., snippet, file,
 package, vulnerability) or VulnAssessmentRelationship being referenced.
 
-This might or might not be different from the originating distribution source
+This can or can not be different from the originating distribution source
```

Comment:
This one reads a bit off - should have further review. Is may or may not allowed?  If so, it may read better.